telegram/request/_httpxrequest.py

```diff
@@ -198,6 +198,16 @@ def http_version(self) -> str:
         """
         return self._http_version
 
+    @property
+    def read_timeout(self) -> Optional[float]:
+        """See :attr:`BaseRequest.read_timeout`.
+
+        Returns:
+            :obj:`float` | :obj:`None`: The default read timeout in seconds as passed to
+                :paramref:`HTTPXRequest.read_timeout`.
+        """
+        return self._client.timeout.read
```

Comment:
doesn't this contradict the docs of BaseRequest.read_timeout? We are returning the timeout even when `read_timeout`  _is_ passed. We should just return `5`?

Reply:
No, what the docs of BaseRequest.read_timeout mean is that this property should be used within `BaseRequest.post` if the `read_timeout` of *that method* was not explicitly specified. For `HTTPXRequest`, that value should be the one specified on init. I elaborated the docs of `BaseRequest.read_timeout` a bit  for this :)